### DISCUSSION TITLE
Relation manager - Action button

### DISCUSSION THREAD

**Discussion by matijarezek** [filamentphp/filament#10538] [Help] [answered]
I have a problem on my TicketCommentsRelationManager.

I have a button in the header that is used to create new comment, but I want it hidden if a ticket has a status 'Closed'.

I wrote it like this, but no matter what is the status the button is visible.

`->headerActions([
                Tables\Actions\CreateAction::make()
                    ->authorize(true)
                    ->hidden(fn (Ticket $ticket): bool => $ticket->isClosed())
                    ->label('New comment')
                    ->disableCreateAnother(),
            ])`

**Comment by leandrocfe** [comment, marked as answer]
https://filamentphp.com/docs/3.x/panels/resources/relation-managers#accessing-the-relationships-owner-record

```php
->headerActions([
    Tables\Actions\CreateAction::make()->hidden(
        fn (RelationManager $livewire): bool => $livewire->getOwnerRecord()->isClosed()
    ),
])
```

**Reply by matijarezek** [reply]
That's it! Thank you so much.
